internal/graphengine/engine_v3_reverse_proxy.go

```diff
@@ -9,10 +9,10 @@ import (
 
 	"github.com/jensneuse/abstractlogger"
 
-	"github.com/TykTechnologies/graphql-go-tools/v2/pkg/engine/datasource/httpclient"
-	"github.com/TykTechnologies/graphql-go-tools/v2/pkg/graphql"
-	"github.com/TykTechnologies/graphql-go-tools/v2/pkg/subscription"
-	gqlwebsocketV2 "github.com/TykTechnologies/graphql-go-tools/v2/pkg/subscription/websocket"
+	httpclientv2 "github.com/TykTechnologies/graphql-go-tools/v2/pkg/engine/datasource/httpclient"
```

Comment:
these packages are already on v2, do we need v2 suffix in the package name alias?

Reply:
because of conflicts in package scope, there's also a v1